sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/Flowlet.cs

```diff
@@ -39,24 +39,20 @@ public Flowlet()
         /// describing the data flow.</param>
         /// <param name="folder">The folder that this data flow is in. If not
         /// specified, Data flow will appear at the root level.</param>
-        /// <param name="additionalProperties">Unmatched properties from the
-        /// message are deserialized this collection</param>
         /// <param name="sources">List of sources in Flowlet.</param>
         /// <param name="sinks">List of sinks in Flowlet.</param>
         /// <param name="transformations">List of transformations in
         /// Flowlet.</param>
         /// <param name="script">Flowlet script.</param>
         /// <param name="scriptLines">Flowlet script lines.</param>
-        public Flowlet(string description = default(string), IList<object> annotations = default(IList<object>), DataFlowFolder folder = default(DataFlowFolder), IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), IList<DataFlowSource> sources = default(IList<DataFlowSource>), IList<DataFlowSink> sinks = default(IList<DataFlowSink>), IList<Transformation> transformations = default(IList<Transformation>), string script = default(string), IList<string> scriptLines = default(IList<string>), object additionalProperties1 = default(object))
+        public Flowlet(string description = default(string), IList<object> annotations = default(IList<object>), DataFlowFolder folder = default(DataFlowFolder), IList<DataFlowSource> sources = default(IList<DataFlowSource>), IList<DataFlowSink> sinks = default(IList<DataFlowSink>), IList<Transformation> transformations = default(IList<Transformation>), string script = default(string), IList<string> scriptLines = default(IList<string>))
             : base(description, annotations, folder)
```

Comment:
This is a breaking change, so please bump the major version not the minor version

Reply:
updated, please review again